app/customer.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import annotations
+
+from app.car import Car
+from app.shop import Shops
+
+
+class Customers:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: Car
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    @staticmethod
+    def add_customer(customers: dict) -> list[Customers]:
+        customers_list = []
+        for customer in customers:
+            customer_object = Customers(
+                name=customer["name"],
+                product_cart=customer["product_cart"],
+                location=customer["location"],
+                money=customer["money"],
+                car=Car(
+                    brand=customer["car"]["brand"],
+                    fuel_consumption=customer["car"]["fuel_consumption"]
+                )
+            )
+            customers_list.append(customer_object)
+        return customers_list
+
+    @staticmethod
+    def count_cost(

```

Comment:
this function is really long. maybe it can be divided into several?

Reply:
Still really long. Find some parts could be take out to another